src/style.css

```diff
@@ -0,0 +1,41 @@
+body {
+  margin: 0;
+  box-sizing: border-box;
```

Comment:
Move this to `*` selector

Reply:
Move only box-sizing to * selector, margin: 0 you can move back to body